paddle/framework/executor.cc

```diff
@@ -130,6 +134,12 @@ void Executor::Run(const ProgramDesc& pdesc, Scope* scope, int block_id,
   if (create_vars && create_local_scope) {
     scope->DeleteScope(local_scope);
   }
+#ifdef FLUID_MEMORY_BENCHMARK
+  VLOG(2) << "-------------------------------------------------------";
+  VLOG(2) << "Memory used after deleting local scope: "
+          << memory::memory_usage(place_);
+  VLOG(2) << "-------------------------------------------------------";
+#endif
```

Comment:
We can just add this as a gflags, not a macro.

Reply:
Done